drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1451,6 +1457,16 @@ function DragAndDropBlock(runtime, element, configuration) {
         return key === TAB;
     };
 
+    var isBrowsingBackwardsInAccessibility = function(evt) {
```

Comment:
I think this name would be better as `isTabbingForwards` or `isTabWithoutShift`, there is nothing accessibility-specific about these functions.

Reply:
Accessibility mode in drag and drop refers to users browsing and interacting with the items and drop zones using the keyboard only. This includes the mentioned user interaction.

Reply:
But I'd also prefer `isTabbingForwards` nonetheless. (Y)

Reply:
An important part of every review we do is to check for accessibility concerns, and using it via a keyboard is definitely one of them. However, my point here is that accessibility isn't a separate mode, anyone can just press tab and use the keyboard to browse. A lot of power users use that as well. 

These functions are just checking if the tab key is pressed and shift is on/off, so that is the function name should imply. Putting accessibility in the name is just confusing because it implies something more than what is going on.